app/models/User.php

```diff
@@ -29,4 +29,9 @@ public function certificateValid($certificate = null)
 
 		return true;
 	}
+
+	public function isAdmin()
+	{
```

Comment:
`$_ENV['PHRAGILE_ADMINS']` is global state usage. It'd be better to have this injected. Same goes for the `$_ENV['PHABRICATOR_URL']` above. Not sure to what extend the framework forces you into this, though it's definitely not good design.